.github/CODEOWNERS

```diff
@@ -1,22 +1,11 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
 
+# Code owners are the final gate for PR approval to their named section of code.
+# If a single PR involves multiple code owners, every code owner should approve
+# a PR prior to merging.
```

Comment:
Do we want to require code owners to review the entire PR or just the portion that they're owning?

Reply:
Just their impacted code seems reasonable